### PR TITLE
Issue #32 - support maxVersion for test cohorts

### PR DESCRIPTION
**Pull request by mkaply**
@leplatrem What do you think of this?

### CODE REVIEW

File: absearch/settings.py [hunk 1]
```diff
@@ -122,6 +122,9 @@ def load(self):
                         # Use a very small version
                         filters['minVersion'] = filters.get('minVersion',
                                                             '0.1')
+                        # Use a very large version
+                        filters['maxVersion'] = filters.get('maxVersion',
+                                                            '1000.0')
```
Comment (leplatrem) [inline absearch/settings.py]: Will fail in 108 years :)

File: data/config.json [hunk 2]
```diff
@@ -201,9 +201,10 @@
             },
             "filters": {
               "sampleRate": 100,
-              "maxSize": 3,
+              "maxSize": 1000,
```
Comment (leplatrem) [inline data/config.json]: would the test fail with 3?

Comment (mkaply) [reply data/config.json]: I left that in accidentally when testing. I'll remove.

**Comment by mkaply** [comment]
This is ready to merge.

**Comment by mkaply** [comment]
So it looks like I do have to bump the cohort size because otherwise the tests will fail (since we're doing more than 3 queries now)

**Comment by mkaply** [comment]
Is there a way to rerun the CI tests? This appears to be a server error.

**Pull request merged**